### DISCUSSION TITLE
[Possible API bug?] - Can't call pronoun in compliment quip.

### DISCUSSION THREAD

**Discussion by zwartkop2** [Monika-After-Story/MonikaModDev#7899] [QA - code and content creation (technical)] [answered]
![screenshot0045](https://user-images.githubusercontent.com/79435778/134589889-e2603d2a-00dd-4707-a963-8df107bd1b6e.png)

**Comment by Booplicate** [comment]
That's not our code, possibly a submod.

**Comment by zwartkop2** [comment]
It's a submod I've been trying to make. I'm still a novice at coding, so I thought I would try to learn by making some custom compliments based on existing code.
I wanted to take the player's pronouns into account in the quips, but then this happened.

**Comment by multimokia** [comment, marked as answer]
Use `renpy.substitute` to preprocess strings you're interpolating into dialogue
